exonum/src/messages/protocol.rs

```diff
@@ -552,6 +560,33 @@ impl TransactionsRequest {
     }
 }
 
+/// Request for pool transactions.
+///
+/// ### Processing
+/// All transactions from mempool are sent to the recipient.
+///
+/// ### Generation
+/// A node can send `PoolTransactionsRequest` during `Status` message
+/// handling.
+#[derive(Clone, PartialEq, Eq, Ord, PartialOrd, Debug, ProtobufConvert)]
+#[exonum(pb = "proto::PoolTransactionsRequest", crate = "crate")]
+pub struct PoolTransactionsRequest {
+    /// Public key of the recipient.
+    to: PublicKey,
```

Comment:
I think that we should use fields of the plain data transfer objects directly without java like wrappers.

Reply:
I suppose it should be done in the epic of pool improvements. But I've created the ticket for this task [ECR-3417](https://jira.bf.local/browse/ECR-3417)

Reply:
I agree. I think we should create a separate task for that.

Reply:
This was been already fixed in `dynamic_services` branch.